otx/api/entities/dataset_item.py

```diff
@@ -493,14 +494,14 @@ def get_metadata_by_name_and_model(self, name: str, model: Optional[ModelEntity]
 
     def wrap(self: T, **kwargs) -> T:
         """Creates a new DatasetItemEntity, overriding only the given arguments to the existing ones for this instance."""
-        return self.__class__(
-            media=kwargs.get("media", self.media),
-            annotation_scene=kwargs.get("annotation_scene", self.annotation_scene),
-            roi=kwargs.get("roi", self.roi),
-            metadata=kwargs.get("metadata", self.get_metadata()),
-            subset=kwargs.get("subset", self.subset),
-            ignored_labels=kwargs.get("ignored_labels", self.ignored_labels),
-        )
+        params = {
```

Comment:
I think metadata can't be passed to new class becuase metadata doesn't have property or setter. Is it ok?

Reply:
Thanks, it should be fixed

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2004/commits/3b1e636fcbc92b0b0fc99b299e82b45d0019862d.